src/main/java/com/simleetag/homework/api/domain/home/member/api/MemberMaintenanceController.java

```diff
@@ -34,18 +31,6 @@ public ResponseEntity<MemberResponse> findOne(@PathVariable Long homeId, @PathVa
         return ResponseEntity.ok(MemberResponse.from(member));
     }
 
-    @Operation(
-            summary = "생성",
-            description = "멤버 생성은 집에 멤버를 추가하는 것과 같습니다."
-    )
-    @PostMapping
-    public ResponseEntity<MemberResponse> join(@PathVariable Long homeId,
-                                               @RequestParam Long userId) {
-        final Home home = homeFinder.findHomeWithMembers(homeId);
-        final Member member = memberService.join(home, userId);
-        return ResponseEntity.ok(MemberResponse.from(member));
-    }
-
     @Operation(summary = "수정")
     @PutMapping("{memberId}")
     public ResponseEntity<MemberResponse> modify(@PathVariable Long homeId,
```

Comment:
HomeId가 PathVariable로 남아있네요.
쿼리스트링으로 변경해야 할 것 같습니다.
위에 조회 메소드도 마찬가지에요!

```suggestion
public ResponseEntity<MemberResponse> modify(@RequestParam Long homeId,
```